charts/tetragon/values.yaml

```diff
@@ -0,0 +1,189 @@
+enabled: true
```

Comment:
We typically reduce the values file to only the deviations from the upstream defaults so it is much more readable.
Are all these values needed?

Reply:
yuck, my bad